examples/01_scale.cpp

```diff
@@ -16,13 +17,16 @@
 #  include <execution>
 #endif
 
-using std::experimental::mdspan;
-using std::experimental::extents;
+namespace MdSpan = MDSPAN_IMPL_STANDARD_NAMESPACE;
+namespace LinearAlgebra = MDSPAN_IMPL_STANDARD_NAMESPACE :: MDSPAN_IMPL_PROPOSED_NAMESPACE :: linalg;
```

Comment:
Obviously not a blocker but why did we not define `MDSPAN_IMPL_PROPOSED_NAMESPACE` to be `Experimental -> Kokkos::Experimental` or `experimental -> std::experimental` so we don't have to nest it with `MDSPAN_IMPL_STANDARD_NAMESPACE`?
Also I can't remember why we decided to use macros rather than alias namespaces.

Reply:
Thanks @dalg24 !  I discussed this question with @crtrott a while back.  I recall that he thought it would be a lot more work to assume that the two namespaces aren't nested.